main.js

```diff
@@ -0,0 +1,101 @@
+
+//Show and Hide The news Sections
+var controlElement = document.querySelector('#show');
+controlElement.addEventListener('click', showHide);
+var content = document.querySelector('#mybutton');
+function showHide(event) {
+    event.preventDefault();
+    if (content.className === "hide") {
+        content.className = "";
+    }
+    else  {
+        content.className = "hide";
+    }
+}
+
+   
+
+
+//Form Selectors //
+
+var userEmail = document.querySelector('#useremail');
+var userName = document.querySelector('#username');
+var userPhone = document.querySelector('#userphone');
+
+var sbumitButton = document.querySelector('#submit-input');
+sbumitButton.addEventListener('click', submitForm);
+
+function submitForm(event) {
+ event.preventDefault();
+    verifyUserEmail();
+    verifyUserName();
+    verifyUserPhone();
+    if (verifyUserEmail() && verifyUserName() && verifyUserPhone()) {
+        userEmail.value = "";
+        userName.value = "";
+        userPhone.value = "";
+        alert('Thanks');
+       
+}
+}
+
+function verifyUserName() {
+    if (userName.value.length <= 0 ) {
+        userName.style.backgroundColor = "red";
+    }
+    else {
+        userName.style.backgroundColor = "white";
+        return true;
+    }
+}
+
+function verifyUserPhone() {
+    if ( userPhone.value.length < 11 && userPhone.value.length > 0 ) {
+        userPhone.style.backgroundColor = "white";
+        return true;
+    }
+    else {
+        userPhone.style.backgroundColor = "red";
+        return false;
+    }
+}
+
+function verifyUserEmail() {
+    if (userEmail.value.length <= 0 || textEmail.value.indexOf('@') < 0 ) {
```

Comment:
this is good but you could also write it like this

```
var isEmailValid = (userEmail.value.length <= 0 || textEmail.value.indexOf('@') < 0);
userEmail.style.backgroundColor = isEmailValid ? 'red' : 'white'; // if you don't understand this code - google ternary operators in JavaScript
return isEmailValid; // this will be false or true
```

This code is necessarily better or worse, but just read it and make sure you understand it (it should be equivalent to your code) and let me know if you have questions about it